### DISCUSSION TITLE
Security Scanning Falscher Text ?

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#561] [Übersetzungsvorschläge]
Wenn ich mit mein Schiff gescannt werde, steht „Halte aktuelle Geschwindigkeit“ vor dem tatsächlichen Scanvorgang, wenn mein Schiff in Bewegung ist. Vielleicht sollte da irgendwas mit halten … oder reduziere Geschwindigkeit stehen. Ich weiß nicht mehr, welches Wort im Englischen Original steht. Könnte dies jemand prüfen? Wer in Bewegung bleibt, der bekommt Crimestat wegen Befehlsverweigerung.

![ScreenShot-2024-07-08_15-54-43-820](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/ccc378cd-d0c8-4652-b9e7-275a57270283)

**Comment by rjcncpt** [comment]
Moin. 

Im englischen steht: Halt Current Velocity
Soweit ich weiß muss man nicht stehenbleiben. Man darf nur nicht wegfliegen.

Grüße und danke!

**Reply by BTSVDanielk** [reply]
Ich war vor 2 Tagen etwas schneller und ich habe Crime bekommen mit dieser Meldung. Dann ist es ein Cig Problem.

**Reply by rjcncpt** [reply]
> Ich war vor 2 Tagen etwas schneller und ich habe Crime bekommen mit dieser Meldung.

Jo. Deshalb soll man die Geschwindigkeit halten. Ich denke das will CIG damit sagen.

**Reply by BTSVDanielk** [reply]
Hatte ich *g

**Reply by rjcncpt** [reply]
Ich habe mal "Geschwindigkeit halten/reduzieren" genommen. Sollte denke ich passen

**Reply by rjcncpt** [reply]
Ich habe mal im Spectrum gefragt

**Reply by BTSVDanielk** [reply]
Mir ist es 2 mal passiert. Heute habe ich nur den Screen gemacht. Dann halte mich auf den laufenden ;)

**Reply by BTSVDanielk** [reply]
![ScreenShot-2024-07-12_14-26-39-598](https://github.com/user-attachments/assets/cfa1603a-4601-4838-9c7c-76cd00596c4c)

![ScreenShot-2024-07-12_14-27-10-ED1](https://github.com/user-attachments/assets/1a7dbd5d-26fc-4612-9a11-28dfd8e0d5ad)

Ich habe jetzt kein Video gemacht, aber ich habe die Geschwindigkeit gehalten. Vielleicht flüchtet man von der Secruity obwohl oben steht "Halte aktuelle Geschwindigkeit".

**Reply by BTSVDanielk** [reply]
Ich bin mir auch sicher das ich vor Tagen mit 22 m/s crime bekommen habe.

**Reply by BTSVDanielk** [reply]
Ich wurde ebend noch einmal gescannt und mit 22 m/s gab es wieder crime.

**Reply by rjcncpt** [reply]
> Ich wurde ebend noch einmal gescannt und mit 22 m/s gab es wieder crime.

Hey. Sorry, man wird leider nicht benachrichtigt bei kommentaren. 

Jo im 3.24 Patch haben wir den Text angepasst :)
